Dockerfile

```
@@ -0,0 +1,33 @@
+# Build stage
+FROM golang:1.24-alpine AS builder
+
+WORKDIR /app
+
+# Install ca-certificates for HTTPS requests
+RUN apk add --no-cache ca-certificates
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)